src/user/dashboard/news-feed/news-feed.scss

```diff
@@ -198,10 +198,6 @@
                 margin-right: 15px;
                 margin-bottom: 15px;
                 .tag-btn {
-                  background: #1a73e8;
-                  border-radius: 15px;
-                  font-family: Inter;
```

Comment:
also, we are having font-family Inter for almost all the texts we are using on donut.

Reply:
Its been removed because that CSS code is redundant now. Our Btn comoponet will always have the following styles 
```
        fontFamily: 'Inter',
        boxShadow: 'none',
        borderRadius: '0.25rem',
        fontStyle: 'normal'
        backgroundColor = '#1A73E8';
        color = '#ffffff';
```
the backgroundColor and color would interchange if outline btnType is selected.

Please go though declaration of the component in `src/common/Button.js`.
Thanks.

Reply:
yup